crates/kubelet/src/bootstrapping/mod.rs

```diff
@@ -66,14 +66,26 @@ async fn bootstrap_auth<K: AsRef<Path>>(
             })?;
         let server = named_cluster.cluster.server;
         trace!(%server, "Identified server information from bootstrap config");
-        let ca_data = named_cluster
-            .cluster
-            .certificate_authority_data
-            .ok_or_else(|| {
-                anyhow::anyhow!(
-                    "Unable to find certificate authority information in bootstrap config"
-                )
-            })?;
+
+        let ca_data = match named_cluster.cluster.certificate_authority {
+            Some(certificate_authority) => base64::encode(
+                read_to_string(certificate_authority)
```

Comment:
Nit: You can use `read` here instead to avoid the string conversion stuff and just get the bytes

Reply:
oh yeah !! can i update and push now ? 
I just forgot it, at first i wanted this to be string before encoding commit.